ns3_adapter/src/ns3_client.cpp

```diff
@@ -13,86 +14,134 @@ NS3Client::~NS3Client() {
     }catch(...){}
 }
 
-bool NS3Client::registermsg(const std::shared_ptr<std::vector<uint8_t>>&message, const std::string &remote_address, unsigned short remote_port,
-                            unsigned short local_port)
-{
-    bool success = false;
-    
-    if (connect(remote_address, remote_port, local_port))
+
+bool NS3Client::connect_registration(const std::string &remote_address, unsigned short remote_port) {
```

Comment:
We are currently never calling this method and there is no documentation to explain how it is different from `connect_registration_and_broadcasting`. What is the importance of the bool we return? If it is success, we do we return false when running_ is true.

Reply:
removed this function